dashboards/ansible-automation-controller/dashboard.json

```diff
@@ -259,7 +275,7 @@
 					"title": "Time Spent Sending Metrics To Other Nodes (seconds)",
```

Comment:
```suggestion
					"title": "Time Spent Sending Metrics to Other Nodes (seconds)",
```

Reply:
Hi @mdumpati ,
As per your comment, I have changed the card title. Kindly review it.

Thank you...